client/src/app/site/assignments/modules/assignment-poll/components/assignment-poll-vote/assignment-poll-vote.component.ts

```diff
@@ -192,7 +192,12 @@ export class AssignmentPollVoteComponent extends BasePollVoteComponent<ViewAssig
             return;
         }
         const title = this.translate.instant(`Submit selection now?`);
-        const content = this.translate.instant(`Your decision cannot be changed afterwards.`);
+        let content = this.translate.instant(`Your decision cannot be changed afterwards.`);
+        if (this.poll.max_votes_amount > 1 && !this.isGlobalOptionSelected()) {
+            content =
+                this.translate.instant(`Your votes: ${this.getVotesCount()}/${this.poll.max_votes_amount}<br>`) +
```

Comment:
Please mark only "your votes" as translateable string.

Reply:
Done in commit https://github.com/OpenSlides/openslides-client/pull/1053/commits/3ba572f71d066229cf9ce9ea5d6109b4a0c04106